### ISSUE TITLE
No page title for the ocaml.org/api/index.html page

### ISSUE THREAD

**Issue by lindaokorie27** [ocaml/v2.ocaml.org#1555]
## Context
I ran several pages  of the site through an accessibility tester and noted the areas of deficiency with the site. The API docs index page does not have as page title. The browser then defaults to showing the url path of the page  and will be read the same way to screen readers. This is not expressive enough and would be bad UX.

![url](https://user-images.githubusercontent.com/38317939/115863185-06d46b00-a42d-11eb-9e99-770af96e7c7c.JPG)

To recreate, visit https://ocaml.org/docs page and click "Ocaml API" button.

A suitable title should be added to the page

**Comment by dldisha** [comment]
I'm interested in working on this.

**Comment by Bhavika-Gianey** [comment]
> ## Context
> I ran several pages of the site through an accessibility tester and noted the areas of deficiency with the site. The API docs index page does not have as page title. The browser then defaults to showing the url path of the page and will be read the same way to screen readers. This is not expressive enough and would be bad UX.
> 
> ![url](https://user-images.githubusercontent.com/38317939/115863185-06d46b00-a42d-11eb-9e99-770af96e7c7c.JPG)
> 
> To recreate, visit https://ocaml.org/docs page and click "Ocaml API" button.
> 
> A suitable title should be added to the page

The same issue also goes for https://ocaml.org/api/compilerlibref/index.html.
